### DISCUSSION TITLE
Vanilla OS 2 Orchid has been released!

### DISCUSSION THREAD

**Discussion by kbdharun** [Vanilla-OS/live-iso#343] [Announcements]
Introducing Vanilla OS 2 Orchid: Over a year of meticulous development has culminated in this complete rewrite of our operating system. Orchid redefines simplicity and performance, providing a seamless and intuitive experience for everyone regardless of whether you're a web surfer, gamer, developer, or designer. Enjoy an out-of-the-box experience that meets all your needs.

**Checkout the Release post**: https://vanillaos.org/blog/article/2024-07-28/vanilla-os-2-orchid---stable-release 

## Changelog

- **ABRoot v2**: Complete rewrite using OCI images for reliable updates, improved support for atomic transactions, system state dumping, changing kernel flags, a built-in method to edit the configuration, ability to generate local images with extra packages (for drivers, codecs, and libraries), ability to rollback, and re-generate the initramfs.
- **Hybrid Debian Base**: Transition from Ubuntu to a hybrid base made of Debian packages and Vib modules for increased flexibility and control over updates and configuration.
- **LVM Thin Provisioning**: Efficient disk space usage with logical volumes, allowing virtual filesystems larger than the available physical storage and supporting the dynamic allocation of space as needed for the two roots.
- **PolKit Policies**: Replaces sudo for secure privileged operations.
- **VSO v2**: Acts as system shell, package manager, and supports Android apps via Waydroid integrated with F-Droid. Improved the tasks automation system to cover infinite scenarios.
- **DEB/APK Sideloading**: Install .deb and .apk packages with Sideload Utility.
- **Apx v2**: Custom environments, supports various package managers, and introduces stacks for replicating environment configurations. Includes package manager mapping, stack creation, and subsystem management. Distrobox updated to version 1.7.2.1.
- **Apx GUI**: Graphical interface for APX management with ease.
- **FsGuard and FsWarn**: Boot-time system integrity checks to ensure system security and reliability.
- **Vanilla Installer**: Uses a reduced GNOME session, a new Albius backend that replaces distinst, new configuration screens, OEM support, and support for encrypting the `/var` partition with LUKS2. Supports manual partitioning and network configuration during installation.
- **First Setup**: Uses a reduced GNOME session, added network configuration, user creation, hostname configuration, allows picking your browser of choice, and improved screens.
- **PRIME Profiles**: A new GUI for switching graphics cards, enhancing hardware compatibility.
- **Vanilla Tools**: Utilities for managing system features: `cur-gpu` to display the GPU in use, nrun to run a command using the NVIDIA GPU, prime-switch to switch PRIME Profiles.
- **Kernel 6.9.8**: Ensures compatibility with the latest devices and peripherals.
- **GNOME 46**: Updated GNOME to version 46, introducing the new pill icon to switch between workspaces plus all the UI and stability improvements.
- **Vib (Vanilla Image Builder)**: Our new OCI recipe system. Recommended method for creating custom and derivative OCI images of Vanilla OS, facilitating modular and scalable system builds. We have created a [template](https://github.com/Vanilla-OS/custom-image) for users to easily create custom images for Vanilla OS.
- **Recovery Mode**: Integrated in the installer with recovery tools like terminal, GParted, and documentation access for system restoration.
- **Prometheus**: Container library/engine for OCI image management, enhancing deployment processes in Albius and ABRoot.
- **Eratosthenes**: Platform for browsing package details from our Debian repositories, providing detailed package information.
- **Atlas**: Platform for browsing our OCI image details, aiding in system transparency.
- **Pico, Core, Desktop Images**: Structured for modularity and flexibility, forming the base of Vanilla OS.
- **VM and NVIDIA Images**: Proposed at installation time if the right hardware is identified.
- **Dev Image**: Can be used in APX, and provides a large set of development libraries/SDKs and tools.
- **Chronos**: Unified documentation server and frontend combining all guides and documentation for easier access and management.
- **Update Feedback and Control**: A new icon now appears on the top right of the screen when an update is being installed, click on it to stop the ongoing operation.
- **New Wallpaper**: Designed by community member hrstwn.
- **Build Provenance**: We are attesting our OCI images using GitHub Actions to ensure build provenance.
- **New Branding**: Introduced a new refreshed brand. Plymouth updated using our new brand.
- **Deprecation of Vanilla Control Center**: We have deprecated Vanilla Control Center and implemented all our settings in GNOME Settings.
- **Updated AdwDialog Utility**: With many UI and UX improvements.
- **New App Tour**: Introduced a new Tour app that shows after the first setup process.
- **New Differ Utility**: Used to extract info between OCI images.

## Alternative Download

https://download.vanillaos.org/latest.zip

<hr /><em>This discussion was created from the release <a href='https://github.com/Vanilla-OS/live-iso/releases/tag/2.0'>Vanilla OS 2 Orchid</a>.</em>

**Comment by Barlafuss** [comment]
is Orchid beta 2 update automatically to stable ?
Or the system must be completed re-installed with the new iso ? No info found in this article

**Reply by taukakao** [reply]
From the beta, you can automatically upgrade to the stable version.
If it's not doing that for some reason, try `abroot upgrade -f`.

**Reply by Bohne13** [reply]
how to know if I'm using stable or beta?
Is there something like a build number I can compare to the latest build on github / website / etc.?

**Reply by taukakao** [reply]
@Bohne13 
While booting, it should either say "Beta" or "Orchid", next to the Vanilla logo at the bottom.

**Reply by kbdharun** [reply]
@Bohne13 Additionally, if you go to Settings and under System -> About you will see it mention Vanilla OS 2 in regular installs.

**Reply by Barlafuss** [reply]
> From the beta, you can automatically upgrade to the stable version. If it's not doing that for some reason, try `abroot upgrade -f`.
Thanks @taukakao and @kbdharun for the exhaustive answers, I can see I'm on 2.0 now.
:-)

**Comment by ritz123** [comment]
I see the file "latest.zip" is password protected. What is the password to unzip it?

**Reply by kbdharun** [reply]
The Zip file isn't password protected, maybe your download didn't complete successfully leading to this. Alternatively, you can download the ISO file directly from the release.

**Comment by sbe-arg** [comment]
Tried to install fresh on 2 lenovo laptops t41 and t430s install hangs and never finishes when running encryption. Not sure if is just me.

**Reply by sbe-arg** [reply]
The problem seems to be that if you are running on a slow connection it auto suspends and then the downloads don't resume. Having to continously move the mouse around preventing sleep makes it go way further.

Would be good to not have sleep/suspend enabled in the installer.

**Reply by taukakao** [reply]
@sbe-arg 
This issue was fixed very recently. Did you use the latest version of the installer ISO?

**Reply by sbe-arg** [reply]
Yes used the lastest ISO.

**Reply by taukakao** [reply]
@sbe-arg 
I'll need to test it again then. Thanks for bringing it to our attention.

**Reply by taukakao** [reply]
Hm, doesn't seem like it.
I let my Laptop run for quite a while. It turned of the screen but didn't go to sleep.
The setting is also disabled.

**Reply by sbe-arg** [reply]
Ill try again one more time tomorrow. But was odd because i happened on both devices within a few hours.

**Comment by hierro59** [comment]
Not working https://download.vanillaos.org/latest.zip
![image](https://github.com/user-attachments/assets/58ca4aaa-1ac3-44b9-8f55-137ee50c42db)

**Reply by taukakao** [reply]
Can't reproduce this error.
Works fine for me.
Maybe your ISP is blocking vanillaos.

**Reply by hierro59** [reply]
In fact, it is blocked for Venezuela.

**Reply by taukakao** [reply]
Sadly, we can't really help that. 
You will have to use a VPN or download using a torrent. 

<details>
<summary>Here is the torrent link</summary>
magnet:?xt=urn:btih:7768080a31d6d367dac8d55a9aa73f4499540133&dn=VanillaOS-2-Orchid-28jul2024.zip&tr=udp%3A%2F%2Fexplodie.org%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337%2Fannounce&tr=udp%3A%2F%2Fexodus.desync.com%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.internetwarriors.net%3A1337%2Fannounce&tr=udp%3A%2F%2Ftracker.pirateparty.gr%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.tiny-vps.com%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.openbittorrent.com%3A80%2Fannounce&tr=udp%3A%2F%2F9.rarbg.to%3A2710%2Fannounce&tr=udp%3A%2F%2Ftracker.cyberia.is%3A6969%2Fannounce&tr=udp%3A%2F%2Fopen.demonii.com%3A1337%2Fannounce&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969%2Fannounce&tr=udp%3A%2F%2Fopen.stealth.si%3A80%2Fannounce&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969%2Fannounce
</details>

**Comment by V-SCHULTZ** [comment]
Hello,
Just tried it today, I was able to complete the installation (I didn't try disk encryption), and created my admin account (countA).
However, when I created a second linux account (countB), the initial setup wizard popped again, no way to skip it, forced me to create another account (countC) in the process, re-install all basic software for countB (asking me like 50 times the admin password during install), and the wizard still popped after last restart...
So there might be a bug in the wizard script.
I am not yet familiar with atomic distro, however, I would have expected system wide install of basic / initial software, or at least an option to do it.
Thank you for your hard work !

**Reply by taukakao** [reply]
We're currently testing changes that should fix first-setup for additional users. It will be available soon.

Also, you can request the feature to install system-wide here if you want:
https://github.com/Vanilla-OS/first-setup/issues

**Comment by otonon** [comment]
Requires active internet connection to install. Can't skip.
![-5287396666464395345_121](https://github.com/user-attachments/assets/a65f9814-1238-4abe-8e76-9a45af3184d4)

**Reply by taukakao** [reply]
Yes, it needs to download the OS.

**Reply by rotopenguin** [reply]
I tried installing on a new laptop with some horrible Mediatek chip, and got stuck at the "active internet connection is required" screen. I found a USB wifi adapter, plugged it in, _used the gnome-shell quick settings to get on the network_, and the installer does not change. I can go back and forward to the internet page, and it's still just "an active internet connection is required". The installer should probably have a "try again" on that page.

**Reply by KaKi87** [reply]
> Yes, it needs to download the OS.

So the 1.62 GB file doesn't contain the OS ?

How much more is there to download ?

Also, what are the differences between `20240805` and `20240824` ?

Thanks

**Reply by taukakao** [reply]
@KaKi87 
Yes, the ISO contains only a debian install with the vanilla installer.
It needs to download 7GB of data. They are compressed tho, so it's probably quite a bit less to actually download. 

The difference between the ISOs is the installer version. Nothing critical usually.

**Reply by KaKi87** [reply]
> It needs to download 7GB of data. They are compressed tho, so it's probably quite a bit less to actually download.

Download from where exactly ? What's the real download size ? How to download manually onto the USB so that the installer would work offline ?

**Reply by kbdharun** [reply]
> > It needs to download 7GB of data. They are compressed tho, so it's probably quite a bit less to actually download.
> 
> Download from where exactly ? What's the real download size ? How to download manually onto the USB so that the installer would work offline ?

Hi, Vanilla OS doesn't have or support an offline installation currently and you need an active internet connection to install it. The ISO is 1.7 GB and is pretty barebones with just our installer, some disk partitioning and basic tools.

The installer pulls the latest stable OCI Docker images hosted here at the GitHub container registry https://github.com/orgs/Vanilla-OS/packages and depending on the image it might pull 6-7 GB and as mentioned above it might be a bit less since the image is compressed.

**Comment by maxed4901** [comment]
Waiting for dual boot configs

**Reply by ArjixWasTaken** [reply]
~~does it not dual boot if you install to a different drive?~~

you can easily dual boot, you need to edit the real `/etc/default/grub` using a host shell
although, it really is not recommended to dual boot

**Comment by RafalRK75** [comment]
Can't install. GPU error
i9 11900
Radeon 7700XT
![20240805_212556](https://github.com/user-attachments/assets/21fa0d65-30f4-4d8d-a2c2-e7695d5528cc)

**Comment by amal-chandran** [comment]
The settings app is hanging in vmware pro guest. When opens the RDP settings

**Comment by iamdanielsuresh** [comment]
Is this project dead?

**Reply by ArjixWasTaken** [reply]
dunno about dead, but it is certainly experimental, so even if it's alive I would recommend waiting for them to iron out all the kinks

**Reply by kbdharun** [reply]
> Is this project dead?

Hi, the Vanilla OS project is still being actively developed and follows a non-fixed update schedule.

The reason there hasn't been a stable image update since February 26th is because unlike other distributions, we ship updates to our base images only when there is a major version upstream. Or when we need to ship a security update after thorough testing in our dev tags. All packages that you install from GNOME Software (and our first setup) are Flatpak's. Also, the packages you install in VSO shell is directly from Debian Sid, so you will continue receiving updates from upstream for them without routine image updates in our side.

We are current working on shipping GNOME 48 soon (since it's available upstream) and we already have it in our testing repository, but we have encountered a major regression causing issues with boot (after update in existing installations of our dev image tags). So, I currently don't have an ETA on the next update, as we would only ship it after working on a fix and mitigation for the issue.

Checkout https://www.reddit.com/r/vanillaos/comments/1js0tiv/comment/mljktmq/ to know more about the recent happenings surrounding Vanilla OS.

**Comment by ianqsong** [comment]
A new comer got an error for `VanillaOS-2-stable.20251114.iso`
```
 ERROR   unknown command "android" for "vso"
```

**Reply by taukakao** [reply]
We had to remove the android functionality since it introduced a big security issue and no one had time to properly implement it.
